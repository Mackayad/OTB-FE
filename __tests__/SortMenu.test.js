import { shallow } from 'enzyme';
import SortMenu from "../components/sort/sort-menu"

describe("Sort menu component", () => {

  let wrapper

  beforeEach(() => {
    wrapper = shallow(<SortMenu activeSort={1} onSortChanged={jest.fn()} reverse={[false, false, false]} />)
  });

  it('Should render correctly', () => expect(wrapper).toMatchSnapshot())
  
})