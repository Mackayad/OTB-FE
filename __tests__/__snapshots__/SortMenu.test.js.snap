// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sort menu component Should render correctly 1`] = `ShallowWrapper {}`;
